app.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const app = express();
+const healthcheck = require("./controllers/healthcheck.controller");
+const { getTopics } = require("./controllers/topics.controller");
+// const {} = require("./controllers/articles.controller");
+// const {} = require("./controllers/comments.controller");
+// const {} = require("./controllers/users.controller");
+
+app.use(express.json());
```

Comment:
^^ this piece of code is not relevant unless you are passing a body. Please remove it until you need it